README.md

```diff
@@ -0,0 +1,106 @@
+# PersonalFinanceTool
+
+## Introduction
+
+The PersonalFinanceTool (PFT) is an application which allows a user to keep track of the total amount of money they have available as well as bills which they wish to track. Bills will be tracked in the form of name, amount due, and due date. The user can allocate funds from their total amount available to their tracked bills.
+
+## Prerequisities
+
+Before attempting to utilize the PFT, a user should ensure they have the following installed on their machine:
+
+- gcc/g++
+- Qt installation granting access to Qmake executable
+- Add the path to the Qmake executable to PATH environment variable
```

Comment:
This is Mac or Linux specific. Add more clear instructions for Mac, Linux, and Windows.

Reply:
Added more specific instructions for prerequisites section.